### PR TITLE
Sanju github002 patch 1

### PR DESCRIPTION
**Pull request by Sanju-github002**
#### Fixes #(67)

create Forget Password option in log in page.

### CODE REVIEW

**Comment by QAZIMAAZARSHAD** [comment]
Always mention issue number like this #67.

Also share the website link so I can review your changes.

**Comment by satyampgt4** [comment]
@Sanju-github002 are you able to see sign up button in your system
because in my system it is looking like this
image

**Comment by QAZIMAAZARSHAD** [comment]
Yes.

Correct it.

File: login.html [hunk 1]
```diff
@@ -91,11 +108,14 @@
 <div>
 <input type="password" name="password" required placeholder="Password">
 </div>
+<div class="fp">
+<a class="a" href="login.html">Forget Password</a>
+</div
```
Comment (satyampgt4) [inline login.html:113]: ` > `    is missing

File: login.html [hunk 2]
```diff
@@ -91,11 +108,14 @@
 <div>
 <input type="password" name="password" required placeholder="Password">
 </div>
+<div class="fp">
+<a class="a" href="login.html">Forget Password</a>
```
Comment (satyampgt4) [inline login.html:112]: move Forget Password under sign up and reset not above them

**Review by satyampgt4 — CHANGES_REQUESTED** [review]

File: login.html (hunk 1 above)
Comment (Sanju-github002) [reply login.html:113]: Yes sir I change it

File: login.html (hunk 2 above)
Comment (Sanju-github002) [reply login.html:112]: ok

**Comment by Sanju-github002** [comment]
Screenshot 2021-07-12 153241


now check it sir

**Comment by satyampgt4** [comment]
@Sanju-github002 commit and push you it again

**Comment by Sanju-github002** [comment]
> @Sanju-github002 commit and push it again
okk sir

**Pull request closed without merging**